### ISSUE TITLE
Support configuration of access log format

### ISSUE THREAD

**Issue by gtcampbell** [BuoyantIO/linkerd#294]
We currently hard-code the format for access logs; it would be better if we allowed this to be configured. It probably makes the most sense to support Apache's [format strings](https://httpd.apache.org/docs/2.4/mod/mod_log_config.html#formats).

**Comment by leozc** [comment]
Do we have any plan on this ? 

May be parsing a param from route down to Finagle that will help  ?

**Comment by wmorgan** [comment]
It's not on the short-term roadmap AFAIK. Are there specific formats you want to support?

**Comment by bashofmann** [comment]
It would be great to include the time taken to serve the request (like the %{UNIT}T flag in apache's format strings)

**Comment by wmorgan** [comment]
@bashofmann good idea! FYI, you can get aggregate statistics of that info through the metrics.json endpoint currently (and raw data in individual traces).
